packages/@sanity/cli/src/cli.ts

```diff
@@ -240,3 +242,21 @@ function loadAndSetEnvFromDotEnvFiles({workDir, cmd}: {workDir: string; cmd: str
   process.env = {...process.env, ...studioEnv}
   /* eslint-disable no-process-env */
 }
+
+/**
+ * Apparently, Windows environment variables are supposed to be case-insensitive,
+ * (https://nodejs.org/api/process.html#processenv). However, it seems they are not?
+ * `process.env.SYSTEMROOT` is sometimes `undefined`, whereas `process.env.SystemRoot` is _NOT_.
+ *
+ * The `open` npm module uses the former to open a browser on Powershell, and Sindre seems
+ * unwilling to fix it (https://github.com/sindresorhus/open/pull/299#issuecomment-1447587598),
```

Comment:
That's not what I said. I said I wanted to see an issue posted on Node.js first, so it can be fixed properly eventually. When that's done, I'm happy to merge the workaround.